src/main/java/mate/jdbc/dao/CarDaoImpl.java

```diff
@@ -0,0 +1,191 @@
+        return cars;
+    }
+
+    private void deleteDriversFromDb(Car car) {
+        String query = "DELETE FROM cars_drivers WHERE car_id = ?";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(query)) {
+            statement.setLong(1, car.getId());
+            statement.executeUpdate();
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't delete drivers from DB by car "
+                    + car + " .", e);
+        }
+    }
+
+    private Car getCar(ResultSet resultSet) throws SQLException {
+        Car car = new Car();
+        car.setModel(resultSet.getString("model"));
+        Long manufacturerId = resultSet.getObject("m.id", Long.class);
+        car.setId(resultSet.getObject("c.id", Long.class));
+        String manufacturerName = resultSet.getString("name");
+        String manufacturerCountry = resultSet.getString("country");
+        car.setManufacturer(new Manufacturer(manufacturerId,
+                manufacturerName, manufacturerCountry));
+        return car;
+    }
+
+    private void addDriversToDb(Car car) {
+        String query = "INSERT INTO cars_drivers (car_id, driver_id) VALUES (?, ?)";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(query)) {
+            statement.setLong(1, car.getId());
+            for (Driver driver : car.getDrivers()) {
+                statement.setLong(2, driver.getId());
+                statement.executeUpdate();
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't push drivers of car : "
+                    + car + " to DB", e);
+        }
+    }
+
+    private List<Driver> getDriversFromDb(Car car) {
+        String query = "SELECT id, name, license_number FROM cars_drivers cd JOIN drivers d "
+                + "ON cd.driver_id = d.id WHERE cd.car_id = ? AND d.is_deleted = FALSE";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(query)) {
+            statement.setLong(1, car.getId());
+            ResultSet resultSet = statement.executeQuery();
+            List<Driver> drivers = new ArrayList<>();
+            while (resultSet.next()) {
+                drivers.add(getDriver(resultSet));
+            }
+            return drivers;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't get drivers by car " + car + " .", e);
+        }
+    }
+
+    private Driver getDriver(ResultSet resultSet) throws SQLException {
+        Long id = resultSet.getObject("id", Long.class);
+        String name = resultSet.getString("name");
+        String licenseNumber = resultSet.getString("license_number");
+        return new Driver(id, name, licenseNumber);
+    }
```

Comment:
in getCar u used setters to set the info, so here better also create through the setted and not constructor (i've got same request from mentor)